src/modules/userMethods.ts

```diff
@@ -328,13 +328,19 @@ export const userMethods = {
         return prisma.user.findMany({ where: { memberships: { some: { groupId } }, active: { not: true } } });
     },
 
-    edit: async (data: EditUser): Promise<User> => {
+    edit: async (data: EditUserFields) => {
+        const newOrganization = await prisma.organizationUnit.findUnique({
```

Comment:
Error should be thrown if no newOrganization would be found by data.organizationUnitId